tests/test_tags.py

```diff
@@ -316,3 +317,62 @@ def test_copy_related_resource_tag_validate(self):
             ]
         }
         self.assertRaises(PolicyValidationError, self.load_policy, policy)
+
+    def test_copy_related_resource_tag_multi_ref(self):
+        session_factory = self.replay_flight_data('test_copy_related_resource_tag_multi_ref')
+        client = session_factory().client('ec2')
+
+        result = client.describe_instances()
+        self.assertEqual(len(result['Reservations']), 1)
```

Comment:
this call and test feels like it could just be dropped, doesn't really feel like its adding anything except lines and extra api call. you can just assert the tag values on the enis below post policy exec.